src/Native/Bootstrap/main.cpp

```diff
@@ -372,6 +372,11 @@ extern "C" void RhpBulkWriteBarrier()
 {
     throw 42;
 }
+
+extern "C" Object* RhNewArrayAsString(MethodTable*)
```

Comment:
This is referencing `System_Private_CoreLib::System::Runtime::RuntimeImports::RhNewArrayAsString`, but you are adding `RhNewArrayAsString` (without name space). You should see implementation for `System_Private_CoreLib::System::Runtime::RuntimeImports::RhNewArrayAsString` elsewhere in the C++ file.